src/main/kotlin/org/patternfly/Bem.kt

```diff
@@ -26,6 +28,27 @@ public fun String.layout(vararg elements: String): String = combine("pf-l", this
  */
 public fun String.modifier(): String = "pf-m-$this"
 
+/**
+ * Creates a PatternFly modifier CSS class starting with `pf-m-text-`.
+ *
+ * @receiver the modifier name
+ */
+public fun String.text(): String = "pf-m-text-$this"
+
+/**
+ * Creates a PatternFly modifier CSS class starting with `pf-m-height-`.
+ *
+ * @receiver the modifier name
+ */
+public fun String.height(): String = "pf-m-height-$this"
+
+/**
+ * Creates a PatternFly modifier CSS class starting with `pf-m-width-`.
+ *
+ * @receiver the modifier name
+ */
+public fun String.width(): String = "pf-m-width-$this"
```

Comment:
I'm not sure if we really need these String extension functions: `String.text()`, `String.height()` and `String.width()`.  I prefer to be conservative when adding new extension functions. Especially when the receiver is `String`.

Instead I suggest to inline these three function in their enums. Seems that they are not used elsewhere anyway.

Reply:
Good point. I have inlined these, but kept the `String.modifier()`.